plugins/graphiql/src/plugin.ts

```diff
@@ -33,3 +39,10 @@ export const plugin = createPlugin({
     ),
   ],
 });
+
+export const GraphiQLPage = plugin.provide(
+  createRoutableExtension({
+    component: Component,
```

Comment:
i guess this naming clash will keep happening, can we do like `component: import('./components').GraphiQLPage`? ;)